hdfs-commons/src/main/java/org/apache/mesos/hdfs/file/FileUtils.java

```diff
@@ -26,14 +27,20 @@ public static void createDir(File dataDir) {
   /**
    * Delete a file or directory.
    */
-  public static boolean deleteFile(File fileToDelete) {
-    if (fileToDelete.isDirectory()) {
-      String[] entries = fileToDelete.list();
-      for (String entry : entries) {
-        File childFile = new File(fileToDelete.getPath(), entry);
-        deleteFile(childFile);
+  public static boolean deleteDirectory(File fileToDelete) {
+    boolean deleted = false;
+
+    try {
+      if (fileToDelete.isDirectory()) {
```

Comment:
hm, this is the nit anyway. I could go either way, although I would prefer to rename it to something more specific than FileUtils, maybe in the future we will add more custom functionality. Your choice :)